client/src/legacy/LeftAndMain.Tree.js

```diff
@@ -391,11 +393,12 @@ $.entwine('ss.tree', function($){
 
       if(blocking && this.getIsUpdatingTree()) return;
 
-      var self = this, i, includesNewNode = false;
+      var self = this;
       this.setIsUpdatingTree(true);
       self.jstree('save_selected');
 
       // TODO 'initially_opened' config doesn't apply here
+      self.jstree('open_node', this.getNodeByID(0));
```

Comment:
This ensures the root node (the site name) is open, so the tree is visible while any missing nodes are AJAX loaded in